wordpress-mobile/WordPress-Android
# Support fragment migration: Post list section

This PR:
- introduces use of`android.app.v4.support.Fragment` in the `Posts` section of the app.
- removes `WPAlertDialogFragment` as it was not being used.


To test:
1. tap on Blog Posts
2. the `PostsListActivity` along with `PostsListFragment` gets shown. Verify it displays correctly, and device rotation works well.
3. turn airplane mode ON.
4. start a new draft, and write something
5. tap the back arrow, and verify there's a new post with `Local draft` status
6. the `Preview` button should appear in the posts list.
7. Tap `Preview`  and it'll show the `PostPreviewFragment`. Verify your post is displayed.
